server/account/account.go

```diff
@@ -102,4 +102,7 @@ const (
 	// PreimageReveal means an account failed to respond with a valid preimage
 	// for their order during epoch processing.
 	PreimageReveal
+	// Admin means an account was banned by the operator through the admin
+	// interface.
+	Admin
```

Comment:
I couldn't think of a better name.

Reply:
I think adding an arbitrary rule is a bad idea. A client should not be banned for unspecified reasons.